src/Hive.IO/Hive.IO/GhDistributors/GhDistSIA.cs

```diff
@@ -63,14 +61,15 @@ protected override void SolveInstance(IGH_DataAccess DA)
                 {
                     // TO DO: make check that it's not a void
                     zoneAreas[i] += floor.Area;
+                    // TODO no floor on surfaces?
```

Comment:
check this

Reply:
seems there was a decision (from the comments) that only surfaces exposed to the environment (not ground) should be part of the surface areas/types passed to sia380. Is this based on the SIA 380 calculation or some other design decision?

Reply:
yeah that was for determining which surfaces are relevant for transmission heat losses (Q_T). Surfaces to ground are not according to sia380. But if we had an overhang/cantilever and a room above, that floor surface would be actually exposed to the environment and thus relevant for Q_T. OK I have an idea... lemme try sth

Reply:
![image](https://user-images.githubusercontent.com/21927657/128647487-ae37ec8a-986e-4a79-9ba4-3fd3b912b13a.png)

these geometries should now work. overhangs & cantilevers will be interpreted as floor area that accounts to <img src="https://render.githubusercontent.com/render/math?math=Q_T">